pkgs/ffigen/test/native_objc_test/ref_count_test.dart

```diff
@@ -63,22 +63,22 @@ void main() {
       final obj1raw = obj1.ref.pointer;
       final obj2raw = obj2.ref.pointer;
 
-      expect(objectRetainCount(obj1raw), 1);
-      expect(objectRetainCount(obj2raw), 1);
+      expect(objectRetainCount(obj1raw), greaterThan(0));
```

Comment:
Why are we changing these to a generic greaterThan(0)? The tests are now weaker as a result?